artworks/aitagger.go

```diff
@@ -0,0 +1,37 @@
+package artworks
+
+import (
+	"strings"
+)
+
+type AITagger struct{}
+
+func (ai AITagger) AITag(tags []string) bool {
+	aiTags := []string{
+		"aiart",
+		"aigenerated",
+		"aiイラスト",
+		"createdwithai",
+		"dall-e",
+		"midjourney",
+		"stablediffusion",
+	}
```

Comment:
Are we sure that we want to use the same tags for every platform? Just curious if there's any difference between them.

Reply:
It can probably stay like this for now, if need be it can be changed later.